packages/client/components/TopBarSearch.tsx

```diff
@@ -19,6 +19,12 @@ const getShowSearch = (location: NonNullable<RouteProps['location']>) => {
       path: '/team/:teamId',
       exact: true,
       strict: true
+    }) ||
```

Comment:
Not sure how to test if the trailing slash is present or not with less repetition